config/config.example.toml

```diff
@@ -745,4 +745,8 @@ delete_token_url= ""          # base url to delete token from token service
 check_token_status_url= ""    # base url to check token status from token service
 
 [network_tokenization_supported_connectors]
-connector_list = "cybersource" # Supported connectors for network tokenization
\ No newline at end of file
+connector_list = "cybersource" # Supported connectors for network tokenization
+
+[grpc_client.dynamic_routing]
+host = "localhost" 
+port = 7000
```

Comment:
Please add documentation comments for this. Please do the same changes in `config/deployments/env_specific.toml` as well, if not done already.